.github/workflows/cli-beta-publish.yaml

```diff
@@ -0,0 +1,47 @@
+name: CLI beta Publish
+on:
+  pull_request:
```

Comment:
I think you should do on `push` else it might only trigger the first time when you create a PR